lib/qesapdeployment.pm

```diff
@@ -2136,6 +2169,11 @@ sub qesap_terraform_clean_up_retry {
     Return 0: we manage the failure properly
     Return 1: something went wrong or we do not know what to do with the failure
 
+=over 1
+
+=item B<ERROR_LOG> - error log filename
```

Comment:
argument added in  https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18159/  but not documented